rest/taskana-rest-spring/src/main/java/pro/taskana/workbasket/rest/assembler/DistributionTargetRepresentationModelAssembler.java

```diff
@@ -1,21 +1 @@
-package pro.taskana.workbasket.rest.assembler;
```

Comment:
We need to properly delete this class.

Reply:
What do you mean? This class is already deleted